### ISSUE TITLE
Missing tag information for fetchDepositAddress on Bittrex (currency: BURST)

### ISSUE THREAD

**Issue by CannedSurprise** [ccxt/ccxt#4663]
Hi,

When using the fetchDepositAddress method on Bittrex for the currency BURST, both an address and a tag should be returned. However the tag is returned as 'None', and the value that is returned for the address is actually the tag.

- OS: Windows 10
- Programming Language version: Python 3
- CCXT version: 1.18.236
- Exchange: Bittrex
- Method: fetchDepositAddress

Here is the output (tag anonymised):

```
In [9]: bittrex.fetchDepositAddress('BURST')

Request: GET https://bittrex.com/api/v1.1/account/getdepositaddress?apikey=51928ba5ac6e426ea28af59f09cbcf16&nonce=1550083013&currency=BURST {'apisign': '3cd0e662d1d50ec1d04b2ead941e30a4b723178bdd1bd2728598d4228cbcde3e6142192511eae1d55d19c8cf02bfa09a70b040099978b00fb6180a2f2649d9d7', 'User-Agent': 'python-requests/2.18.4', 'Accept-Encoding': 'gzip, deflate'} None

Response: GET https://bittrex.com/api/v1.1/account/getdepositaddress?apikey=51928ba5ac6e426ea28af59f09cbcf16&nonce=1550083013&currency=BURST 200 {'Date': 'Wed, 13 Feb 2019 18:36:53 GMT', 'Content-Type': 'application/json; charset=utf-8', 'Content-Length': '135', 'Connection': 'keep-alive', 'Set-Cookie': '__cfduid=da6d3b2aef6da738a803d1f661ce8a2091550083013; expires=Thu, 13-Feb-20 18:36:53 GMT; path=/; domain=.bittrex.com; HttpOnly', 'Cache-Control': 'private', 'Content-Encoding': 'gzip', 'Strict-Transport-Security': 'max-age=15768000', 'X-XSS-Protection': '1; mode=block', 'X-Content-Type-Options': 'nosniff', 'X-Download-Options': 'noopen', 'Expect-CT': 'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"', 'Server': 'cloudflare', 'CF-RAY': '4a8967348c6fbbd8-LHR'} {"success":true,"message":"","result":{"Currency":"BURST","Address":"463aebabd5d144ac9e60201eba9aa4984c082a88053844bc94a8c05f5477a735"}}

Out[9]:
{'address': 'xxxxxxxxxxxxxxxxxxxxx',
 'currency': 'BURST',
 'info': {'message': '',
  'result': {'Address': 'xxxxxxxxxxxxxxxxxxxxx',
   'Currency': 'BURST'},
  'success': True},
 'tag': None}
```

As you can see the 'tag' is None, and the part where I wrote 'xxxxxxxxxxxxxxxxxxxxx' is where the actual tag is displayed (with the address not displayed anywhere).

Is this a bug or did I do something wrong? Thanks!

**Comment by frosty00** [comment]
It's a bug on the exchange side, (`info` is the raw response from the exchange api)

**Comment by kroitor** [comment]
> It's a bug on the exchange side, (`info` is the raw response from the exchange api)

I don't think it's a bug on the exchange side, it's more of a bug in CCXT, because Bittrex implements base addresses on `exchange.currencies`. I will add a workaround for this shortly.

**Comment by kroitor** [comment]
Fixed in 1.18.238. Let us know if that doesn't help.
